new-log-viewer/src/contexts/StateContextProvider.tsx

```diff
@@ -149,6 +157,13 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
     }, []);
 
     const loadFile = useCallback((fileSrc: FileSrcType, cursor: CursorType) => {
+        // Backup and unset `filePath` in the URL, which can be restored after successful loading
+        // (when `WORKER_RESP_CODE.LOG_FILE_INFO` is received) if `fileSrc` is a URL.
+        filePathRef.current = "string" === typeof fileSrc ?
+            fileSrc :
+            null;
+        updateWindowUrlSearchParams({[SEARCH_PARAM_NAMES.FILE_PATH]: null});
```

Comment:
Hm, this behaviour is a bit odd from a user-perspective:

* If the user has a typo in file path, then on error, the file path is erased making it harder for the user to quickly fix the typo.
* If the file takes a while to load, the file path disappears and then reappears.

Can we somehow erase the file path from the URL when the user drops a file in?

Reply:
True, true

Sure